crates/phactory/api/tests/snapshots/test_block_abi__sync_blocks_abi_should_not_change-2.snap

```diff
@@ -30,35 +30,35 @@ expression: "travel_types::<blocks::SyncParachainHeaderReq>()"
         },
         {
           "name": "Hash",
-          "type": 3
+          "type": null
```

Comment:
It's strange the type changed to `null`.

These snapshots detects whether the data structure of blocks synced into pRuntime changes.

We need to check the compatibility:
- whether the old verions of pRuntime accept the new block data.
- whether the new pRuntime accept block data from old version of chain node.

And we'd better make clear understand on what would these structure changes affect.

Reply:
Looks like this is due to the `#[derive(TypeInfo)]` now trims all zero sized types.